src/main/java/com/alkemy/ong/service/impl/IUserService.java

```diff
@@ -0,0 +1,19 @@
+package com.alkemy.ong.service.impl;
+
+import com.alkemy.ong.repository.UserRepository;
+import com.alkemy.ong.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+@Service
+public class IUserService implements UserService {
+    @Autowired
+    private UserRepository userRepo;
+    @Override
+    public void deleteUserById(String id) {
+
```

Comment:
El espacio este Ani, me parece que está de más.

Reply:
comment the methods at the beginning with what it does

Reply:
Comment added and blank space deleted, thanks fellows